addons/calendar/tests/test_attendees.py

```diff
@@ -131,3 +131,13 @@ def test_default_attendee_2(self):
         })
         self.assertNotIn(self.partner, event.attendee_ids.partner_id, "Partner should not be in attendee")
         self.assertIn(partner_bis, event.attendee_ids.partner_id, "Partner bis should be in attendee")
+
+    def test_not_notify_partner_achived(self):
+        partner_achived = self.env['res.partner'].create({'name': "orga", "active": "false"})
+        event = self.env['calendar.event'].create({
+            'name': "Doom's day",
+            'start': datetime(2019, 10, 25, 8, 0),
+            'stop': datetime(2019, 10, 27, 18, 0),
+            'partner_ids': [(4, partner_achived.id)],
+        })
+        self.assertFalse(event.attendee_ids._send_mail_to_attendees('calendar.calendar_template_meeting_reminder', force_send=True, ignore_recurrence=True))
```

Comment:
Does this actually fail without the change?

If not, you may best be served by a MailCase test class. There already exists TestEventNotification maybe the test should be there?